core/runtime/jsruntime.rs

```diff
@@ -377,6 +380,10 @@ pub struct RuntimeOptions {
   /// executed tries to load modules.
   pub module_loader: Option<Rc<dyn ModuleLoader>>,
 
+  /// Provide a function that may optionally provide a metrics collector
+  /// for a given op.
+  pub op_metrics_fn: Option<OpMetricsFactoryFn>,
```

Comment:
Could you include example either on `OpMetricsFactoryFn` or directly in the docstring for this field?

Reply:
I wanted to hold off on that while we iterate on the API a bit. This may change or evolve and I wanted to plug it into deno itself before committing to the API fully.

Reply:
There's a test in this PR that demonstrates the current version of the API, but I'm sure this will evolve:

https://github.com/denoland/deno_core/pull/244/files#diff-cdffb2ed85028df867198ed8c4cfb966082c88913fb9722eb65bec42aa3de9e3R633